lib/matplotlib/axes/_base.py

```diff
@@ -1645,7 +1645,7 @@ def axis(self, *args, emit=True, **kwargs):
                     self.set_autoscale_on(False)
                     xlim = self.get_xlim()
                     ylim = self.get_ylim()
-                    edge_size = max(np.diff(xlim), np.diff(ylim))
+                    edge_size = max(np.diff(xlim), np.diff(ylim))[0]
```

Comment:
```suggestion
                    edge_size, = max(np.diff(xlim), np.diff(ylim))
```

Will that work instead?

Reply:
And is this a behavior change that is intentional on numpy's part or something we should be reporting back up to them?

Reply:
It would work, but this is merged now...

I've asked [upstream](https://github.com/numpy/numpy/pull/13913#issuecomment-515569002) on the PR that introduced the change.

Reply:
oops, sorry I missed that there was a suggested change in here, and a question.